cognite/client/_api/assets.py

```diff
@@ -55,6 +60,31 @@
     from cognite.client.utils._priority_tpe import PriorityThreadPoolExecutor
 
 
+SortArg: TypeAlias = Union[
```

Comment:
SortSpec is maybe better? or maybe even just Sort?
```suggestion
SortSpec: TypeAlias = Union[
```